README.md

```diff
@@ -941,6 +942,23 @@ replace all occurrences of  `a`  with  `b` <br /> _note:_  this does simple stri
 //=> "Foo Baz"
 ```
 
+### searchString ( [code](https://github.com/nymag/nymag-handlebars/blob/master/helpers/strings/searchString.js) | [tests](https://github.com/nymag/nymag-handlebars/blob/master/helpers/strings/searchString.test.js) )
+
+check if a substring exist within a string. This is very similiar to the<br />indexOf helper, except it returns a boolean.
```

Comment:
Should amend this that it uses `String.prototype.includes()` instead of `indexOf`